Makefile

```diff
@@ -205,21 +205,6 @@ image/build/push: image/build image/push
 
 
 ############ E2E TEST COMMANDS ############
-.PHONY: test/e2e/prow
```

Comment:
would it make more sense to make rhoam a default e2e so that the above line would trigger rhoam tests instead e2e-rhoam?

Reply:
If we only had RHOAM e2e test I would agree with you but as there is also the multitenant-rhoam I think it is best to keep it as is in the Makefile. It is clearer what the make targets do. However it would be possible to update the openshift/release configuration. To have that to be called e2e instant of e2e-rhoam. That is even something that could be done now without this PR being merged.

Reply:
No strong opinion on this so whatever you think is best.